.github/workflows/.tilbakestill.yaml

```diff
@@ -0,0 +1,25 @@
+name: .tilbakestill.yaml
+
+on:
+  workflow_call:
+    inputs:
+      applikasjon:
+        description: 'Navnet på applikasjonen'
+        required: true
+        type: string
+
+env:
+  image: ghcr.io/${{ github.repository }}/etterlatte-${{ inputs.applikasjon }}:main
+
+jobs:
+  tilbakestill:
+    name: tilbakestill-dev
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nais/deploy/actions/deploy@v1
```

Comment:
Burde vi bruke vårt eget deploy-script i steden?

Reply:
Ja, du tenkjer på det som er i .deploy.yaml -> deploy-to-dev-gcp?

Mitt klare svar er _kanskje_. Det vi kan gjera er å leggje på input-flagg der som seier om det er kun deploy til dev (sia eg ser for meg at vi seinare vil ha automatisk deploy til prod), og så leggje inn ifs på jobbane der.

Trur kanskje `RESOURCE: apps/${{ github.workflow }}/.nais/dev.yaml` da vil bli feil, som vi jo kan styre med parameterisering, men magekjensla er at det blir komplekst 🤔

Reply:
Jepp du har nok rett der, den vil få en helt annen flyt da vi kommer til prod og da er det nok fint å ikke blande de to.